pegasus/helpers/section_api_helpers.rb

```diff
@@ -281,7 +281,16 @@ def self.fetch_user_sections(user_id)
        join(:users, :id=>:user_id).
        select(*fields).
        where(user_id:user_id).
-       map{|i| self.new(i).to_hash}
+       map{|i| self.new(i).to_owner_hash}
+  end
+
+  def self.fetch_student_sections(student_id)
+    DASHBOARD_DB[:sections].
+        select(*fields).
+        join(:followers, :section_id=>:id).
+        join(:users, :id=>:student_user_id).
+        where(student_user_id: student_id).
+        map{|i| self.new(i).to_member_hash}
```

Comment:
Now that I look at it I think this would be clearer as `map{|row| ...` instead of `map{|i|` (I realize this is existing code)